docs/_functions/domain/CF_WORKER_ROUTE.md

```diff
@@ -0,0 +1,29 @@
+---
+name: CF_WORKER_ROUTE
+parameters:
+  - pattern
+  - script
+---
+
+`CF_WORKER_ROUTE` uses [Cloudflare Workers](https://developers.cloudflare.com/workers/) 
+API to setup [routes](https://developers.cloudflare.com/workers/platform/routes)
+for a given domain.
+
+If _any_ `CF_WORKER_ROUTE` function is used then `dnscontrol` will manage _all_ 
+Worker Routes for the domain.
+
+WARNING: This interface is not extensively tested. Take precautions such as making
+backups and manually verifying `dnscontrol preview` output before running
+`dnscontrol push`. This is especially true when mixing Worker Routes that are
```

Comment:
Can we mix worker routes that aren't managed by DNSControl?  If not, this last sentence should make that clear.

Reply:
To be frank, this was copied from CF_REDIRECT. The same logic applies here.

I confess I don't like the original text too I don't think anybody would use DNS Control and manually set some routes (since any route manually included will be removed in next push). Or I am missing something?

Reply:
I think I got it. The original text was written that way because there are OTHER kind of page rules not touched by DNSControl (only "Forwarding URL" rules are managed by it).

I'm removing this sentence.